tests/test_triple_refinement.py

```diff
@@ -0,0 +1,21 @@
+from pytest import mark
+
+
+@mark.asyncio
+async def test_triple_refinement(triples_files_prefix):
+    from honeybadgermpc.passive import runProgramInNetwork, generate_test_triples
+    from honeybadgermpc.triple_refinement import refineTriples
+    N, t = 6, 1
+    # FIXME: This needs to be replaced with a fixture
+    generate_test_triples(triples_files_prefix, N, N, t)
```

Comment:
@sbellem How can I use a fixture here? Note: I want to pass different values of `N` and `t` to `triples_shares_files`